src/routing/wrappers/Probe.svelte

```diff
@@ -29,7 +29,7 @@
     if (!$probe) return undefined;
     return getProbeViewType(
       $probe.type,
-      $probe.kind,
+      $probe.info.calculated.latest_history.details.kind,
```

Comment:
GitHub won't allow me to add a comment where I want so...

do all of the *FIXME* comments above still apply? It looks to me like they do, but I thought it would be good to double-check.